PLUGIN_DEV.md

```diff
@@ -25,10 +24,10 @@ If you think we missed exposing a crucial lib or Grafana component let us know b
 ### Deprecated components 
 
 The angular directive `<spectrum-picker>` is now deprecated (will still work for a version more) but we recommend plugin authors
-to upgrade to new `<color-picker color="ctrl.color" onChange="ctrl.onSparklineColorChange"></color-picker>`
+upgrade to new `<color-picker color="ctrl.color" onChange="ctrl.onSparklineColorChange"></color-picker>`
```

Comment:
I'm curious about this change. I would've guessed that:

> ... but we recommend plugin authors _to_ upgrade ...

is more correct than

> ... but we recommend plugin authors upgrade ...

Reply:
The first "to" is unnecessary. The change just tightens up the verbiage and removes and unneeded word. Plus, it sounds better to my ear when spoken aloud.